pyface/ui/null/action/action_item.py

```diff
@@ -92,15 +92,7 @@ def __init__(
         # If the action has an image then convert it to a bitmap (as required
         # by the toolbar).
         if action.image is not None:
-            image = action.image.create_image()
-            path = action.image.absolute_path
-            bmp = image_cache.get_bitmap(path)
-
-        else:
-            from pyface.api import ImageResource
-
-            image = ImageResource("foo")
-            bmp = image.create_bitmap()
+            action.image.create_image()
```

Comment:
why do we need to keep this?

Reply:
I thought there was a reason but when i looked a second time, i couldn't figure out why. It looks like it can be removed and I removed it.